files/sockjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+	"packageManager": "github",
+	"name": "sockjs-client",
+	"repo": "sockjs/sockjs-client",
+	"files": {
+		"basePath": "dist/",
+		"include": ["sockjs.min.js", "sockjs.js"]
```

Comment:
I think we can only include the min file. Can you remove the unminified one please?